assets/terraform-modules/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -193,7 +194,7 @@ storage:
             -v /opt/bootkube/assets:/assets:ro \
             -v /etc/kubernetes:/etc/kubernetes:rw \
             --network=host \
-            quay.io/kinvolk/bootkube:v0.14.0-helm4 \
+            quay.io/kinvolk/bootkube:v0.14.0-helm4-${os_arch} \
```

Comment:
our bootkube also offers multi-arch manifests.

Reply:
Ok, I've removed this and checked that indeed it downloads the right image. I guess it should be removed from the packet code?

Reply:
Yes, but I guess it's out of scope, created an issue for it: #1379